prow/plugins/require-matching-label/require-matching-label.go

```diff
@@ -155,6 +171,22 @@ func handlePullRequest(pc plugins.Agent, pre github.PullRequestEvent) error {
 	}
 	return handle(pc.Logger, pc.GitHubClient, cp, pc.PluginConfig.RequireMatchingLabel, e)
 }
+func handleCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent) error {
+		return handleComment(pc.GitHubClient, pc.Logger, &ce)
+}
+func handleComment(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) error {
+	//consider open and close PRs and new comments
+	if e.IssueState != "open" || e.IssueState != "closed" || e.Action != github.GenericCommentActionCreated {
+		return nil
+	}
+	// Only consider "/check-required-labels" comments.
+	if !checkRequiredLablesRe.MatchString(e.Body) {
+		return nil
+	}
+
+	//to be continued
```

Comment:
I assume you will continue in this PR? Otherwise I don't think we should add help/example for a command that does not do anything (yet).